mantis/acquisition/autotracker.py

```diff
@@ -330,10 +331,15 @@ def estimate_shift(self, ref_img: ArrayLike, mov_img: ArrayLike, **kwargs) -> np
         if not autofocus_method_func:
             raise ValueError(f'Unknown autofocus method: {self.tracking_method}')
 
-        shifts_zyx = autofocus_method_func(ref_img=ref_img, mov_img=mov_img, **kwargs)
+        shifts_zyx_pix = autofocus_method_func(ref_img=ref_img, mov_img=mov_img, **kwargs)
 
         # shifts_zyx in px to shifts_zyx in um
-        self.shifts_zyx = np.array(shifts_zyx) * self.scale
+        shifts_zyx_um = np.array(shifts_zyx_pix) * self.scale
```

Comment:
TODO: make sure `self.scale` is of length 3, and that the Z, Y, X scales are correct. The Z scale changes per experiment and should be fetched or passed from the metadata